Client.xcodeproj/project.pbxproj

```diff
@@ -4564,6 +4579,7 @@
 				E68E7ADC1CAC208200FDCA76 /* SetupPasscodeViewController.swift in Sources */,
 				7B3631EA1C244FEE00D12AF9 /* Theme.swift in Sources */,
 				7BC68D321CC153920043562A /* AppMenuItem.swift in Sources */,
+				A9530A861CCF45FB00F01D0E /* UserScript.swift in Sources */,
```

Comment:
Hi，jhugman, I am totally agree with you for this patch.
But here is another reason to use the name as following:
Because we hope the browser can provide both no image mode and night mode, and  UserScript.swift imports the two scripts to webview as one source, so we think the   UserScript.js can be a file to manage all the scripts like these features, so We push noImageMode and nightMode scripts in this file and named UserScript.js. 
What do you think about it? If you still think NoImageModeHelper.js is better, I can change the name, and try to modify UserScript.swift to import scripts from different files, thanks for your review.